shopify-client/src/views/ShopList.vue

```diff
@@ -242,4 +242,9 @@ input{
   margin: 5px;
   vertical-align: top;
 }
+
+.card-img-top{
```

Comment:
I suggest going to line 63 and replacing the img tag with a b-img tag and using the "fluid" prop, see https://bootstrap-vue.org/docs/components/image#responsive-images

Reply:
PS: Remember that we are using BootstrapVue to make our lives easier, take advantage of it! You don't have to go through the pain of writing CSS when Bootstrap provides you with the resources to simplify styling.

Reply:
i did the changes you asked, but it does not resolve the issue at hand
I will still need some css